spec/models/service_template_transformation_plan_spec.rb

```diff
@@ -276,17 +277,40 @@
       expect(service_template.transformation_mapping).to eq(transformation_mapping)
       expect(service_template.vm_resources.collect(&:resource)).to match_array([vm1, vm2])
       expect(service_template.vm_resources.collect(&:status)).to eq([ServiceResource::STATUS_QUEUED, ServiceResource::STATUS_QUEUED])
-      expect(service_template.vm_resources.find_by(:resource_id => vm1.id).options)
-        .to eq("pre_ansible_playbook_service_template_id" => apst.id, "osp_security_group_id" => security_group1.id, "osp_flavor_id" => flavor1.id)
-      expect(service_template.vm_resources.find_by(:resource_id => vm2.id).options)
-        .to eq("pre_ansible_playbook_service_template_id" => apst.id, "post_ansible_playbook_service_template_id" => apst.id, "osp_security_group_id" => security_group1.id, "osp_flavor_id" => flavor1.id)
+
+      stub_const("VirtualMachine", Class.new(ManageIQ::Providers::InfraManager::Vm) do
```

Comment:
Can you namespace this?  `ManageIQ::Providers::InfraManager::WarmMigrationVm` or something?

Reply:
Done.  Changed `VirtualMachine` to `ManageIQ::Providers::InfraManager::WarmMigrationVm`.